jsonp/1.1/tck_results_summary.txt

```diff
@@ -0,0 +1,1450 @@
+[javatest.batch] Number of Tests Passed      = 169
```

Comment:
The TCK results should be published on a page related to the implementation project,
not here.

Reply:
Done